desktop/aseprite.appdata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">aseprite.desktop</id>  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
```

Comment:
Hi @Mailaender, actually Aseprite is distributed under GPL-2.0, I'm thinking to change it to GPL-3.0 but at this moment it's 2.0.